Client/Frontend/Toolbar+URLBar/ConnectionButton.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+
+final class ConnectionButton: UIButton {
+        
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureView()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - Helpers
+
+extension ConnectionButton {
+    
+    private func configureView() {
+        clipsToBounds = false
+        imageView?.contentMode = .center
+        adjustsImageWhenHighlighted = false
+        imageEdgeInsets = .init(top: 0, left: 10, bottom: 0, right: 0)
+    }
+    
+    private func updateButtonImageAccordingToStatus(_ status: WebsiteConnectionTypeStatus) {
+        setImage(ConnectionStatusImage.getForStatus(status: status), for: .normal)
+    }
+    
+    func updateAppearanceForStatus(_ status: WebsiteConnectionTypeStatus) {
+        updateButtonImageAccordingToStatus(status)
+    }
+    
+    func evaluateNeedingVisbilityForURLScheme(urlScheme: String?) -> Bool {
```

Comment:
💡  There's. a typo in visibility: evaluateNeedingVisbilityForURLScheme -> evaluateNeedingVis**I**bilityForURLScheme

❓   Maybe get rid of one of the `urlScheme`? e.g.

`func evaluateNeedingVisbilityForURLScheme(_ urlScheme: String?) -> Bool {`

or

`func evaluateNeedingVisbility(for urlScheme: String?) -> Bool {`

Reply:
How could I not notice it 🤦 . Thanks a lot for that @kimar! I'll go with the 1st one as that was the original intention